src/controllers/dungeonController.js

```diff
@@ -15,6 +16,19 @@ class DungeonController {
         res.status(err.message).json(errorResponse[err.message]);
       });
   }
+
+  collectReward(req, res) {
+    this.dungeonService.collect(req.headers.heroid)
+      .then(data => res.status(200).json(data))
+      .catch((err) => {
+        const errorResponse = {
+          400: 'Please enter a valid heroId',
+          412: 'Dungeon not finnished yet',

```

Comment:
The finnish are a good people, they drink a lot and like death metal, but I don't think they belong here.

Reply:
corrected